app/src/main/java/com/djinc/edumotive/MainActivity.kt

```diff
@@ -3,41 +3,51 @@ package com.djinc.edumotive
 import android.os.Bundle
 import androidx.activity.ComponentActivity
 import androidx.activity.compose.setContent
-import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.*
 import androidx.compose.material.MaterialTheme
 import androidx.compose.material.Surface
 import androidx.compose.material.Text
 import androidx.compose.runtime.Composable
+import androidx.compose.runtime.mutableStateOf
 import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
 import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.djinc.edumotive.screens.Dashboard
+import com.djinc.edumotive.screens.Exercises
+import com.djinc.edumotive.screens.Main
+import com.djinc.edumotive.screens.Parts
 import com.djinc.edumotive.ui.theme.EdumotiveTheme
+import com.djinc.edumotive.utils.rememberWindowSizeClass
+
+enum class Screens {
+    DASHBOARD, PARTS, EXERCISES
+}
 
 class MainActivity : ComponentActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContent {
+            val windowSize = rememberWindowSizeClass()
             EdumotiveTheme {
-                // A surface container using the 'background' color from the theme
                 Surface(
                     modifier = Modifier.fillMaxSize(),
                     color = MaterialTheme.colors.background
                 ) {
-                    Greeting("Android")
+                    Main(windowSize)
+//                    Row {
```

Comment:
Remove comments?

Reply:
Code is indeed deprecated and has been removed